### ISSUE TITLE
[Bug]: Crash on World Join

### ISSUE THREAD

**Issue by Lukebb04** [TeamLapen/Vampirism#1233]
### Minecraft Version

1.20.x

### Forge Version

47.1.6

### Vampirism Version

1.9.1-beta.1

### Singleplayer/Multiplayer

Singleplayer

### What happened?

The game crashes on world join

### Relevant log output

```shell
Game Output
java.util.concurrent.CompletionException: java.lang.NoSuchMethodError: 'float net.minecraftforge.event.entity.EntityEvent$Size.getOldEyeHeight()'
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:649)
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.util.thread.ProcessorMailbox.m_18759_(ProcessorMailbox.java:91)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.util.thread.ProcessorMailbox.m_18747_(ProcessorMailbox.java:146)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.util.thread.ProcessorMailbox.run(ProcessorMailbox.java:102)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.TickTask.run(TickTask.java:18)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.util.thread.BlockableEventLoop.m_6367_(BlockableEventLoop.java:156)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.util.thread.ReentrantBlockableEventLoop.m_6367_(ReentrantBlockableEventLoop.java:23)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:770)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:161)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.util.thread.BlockableEventLoop.m_7245_(BlockableEventLoop.java:130)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.MinecraftServer.m_129961_(MinecraftServer.java:753)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.MinecraftServer.m_7245_(MinecraftServer.java:747)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.util.thread.BlockableEventLoop.m_18701_(BlockableEventLoop.java:139)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.MinecraftServer.m_130012_(MinecraftServer.java:733)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.MinecraftServer.m_129940_(MinecraftServer.java:459)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.MinecraftServer.m_130006_(MinecraftServer.java:310)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.client.server.IntegratedServer.m_7038_(IntegratedServer.java:63)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:634)
	at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251)
	at java.base/java.lang.Thread.run(Thread.java:833)


Crash Report
java.lang.StackOverflowError: Exception ticking world
	at net.minecraft.world.entity.LivingEntity.m_6380_(LivingEntity.java:3256) ~[client-1.20.1-20230612.114412-srg.jar%23158!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:vampirism.mixins.json:LivingEntityAccessor,pl:mixin:APP:vampirism.mixins.json:MixinLivingEntity,pl:mixin:A}
	at net.minecraft.world.entity.Entity.getEyeHeightAccess(Entity.java:3456) ~[client-1.20.1-20230612.114412-srg.jar%23158!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:vampirism.mixins.json:MixinEntity,pl:mixin:A}
	at net.minecraftforge.common.extensions.IForgeEntity.getEyeHeightForge(IForgeEntity.java:442) ~[forge-1.20.1-47.1.6-universal.jar%23163!/:?] {re:mixin,re:computing_frames,re:classloading}
	at net.minecraft.world.entity.LivingEntity.m_6431_(LivingEntity.java:3260) ~[client-1.20.1-20230612.114412-srg.jar%23158!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:vampirism.mixins.json:LivingEntityAccessor,pl:mixin:APP:vampirism.mixins.json:MixinLivingEntity,pl:mixin:A}
```

### Reproduce steps

1. Load singleplayer world

**Comment by Cheaterpaul** [comment]
This was a bug introduced in forge. Please update to Forge 47.1.29

**Comment by Cheaterpaul** [comment]
fixed by #1231
